wandb/sdk/launch/agent/agent.py

```diff
@@ -64,6 +160,11 @@ def __init__(self, api: Api, config: Dict[str, Any]):
             self._max_jobs = float("inf")
         else:
             self._max_jobs = max_jobs_from_config
+        self._pool = ThreadPool(
+            processes=int(min(64, self._max_jobs)),
```

Comment:
is this a hard cap on the total processes? Can we make this a constant at the top 😁
```suggestion
            processes=int(min(MAX_THREADS, self._max_jobs)),
```